plugins/azure-devops/src/plugin.ts

```diff
@@ -56,3 +56,14 @@ export const EntityAzurePipelinesContent = azureDevOpsPlugin.provide(
     mountPoint: azureDevOpsRouteRef,
   }),
 );
+
+export const EntityAzurePullRequestsContent = azureDevOpsPlugin.provide(
+  createRoutableExtension({
+    name: 'EntityAzurePullRequestsContent',
+    component: () =>
+      import('./components/EntityPageAzurePullRequests').then(
+        m => m.EntityPageAzurePullRequests,
+      ),
+    mountPoint: azureDevOpsRouteRef,
```

Comment:
Does this need to have a different route ref? Also the naming could maybe be slightly more specific.

```
export const azurePipelinesEntityContentRouteRef = createRouteRef({
  title: 'Azure Pipelines Entity Content',
});

export const azurePullRequestsEntityContentRouteRef = createRouteRef({
  title: 'Azure Pull Requests Entity Content',
});
```

Reply:
Yes, I'll make these changes, this part is still a little magical to me. I also like the naming, it's more clear, thanks

Reply:
These changes have been made, did some minor clean up in the `plugin.ts` and `routes.ts` files as well